CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 2.1.4 (TBD)
 
 - Bugfix: Fix race condition that occurred when intercepting a ReplicaSet while another pod was terminating in the same namespace (this fixes a transient test failure)
+- Bugfix: Telepresence no longer require that port 1080 must be available.
```

Comment:
I would say this is a change, since requiring 1080 wasn't really a _bug_ per say

Reply:
+1 to what Donny said